src/appshell/qml/DevTools/Gallery/GeneralComponentsGallery.qml

```diff
@@ -389,6 +390,43 @@ Rectangle {
         }
     }
 
+    Component {
+        id: visibleButtonSample
+
+        Column {
+            spacing: 8
+
+            width: 200
+
+            VisibleButton {
+                text: "Visible Button"
+
+                onVisibleEdited: {
+                    isVisible = !isVisible
+                }
+            }
+
+            VisibleButton {
+                contentItem: RowLayout {
```

Comment:
I think it makes more sense to keep it simple

I mean if I'd want to attach another button to this one, I'd make that layout locally, otherwise the component seems to be overcomplicated

Reply:
Or it's just will be another component who will make that layout for me

Reply:
It will be difficult to reuse in the instruments panel
there is another button between the eye and the text
![image](https://user-images.githubusercontent.com/10116828/124762544-4b9f7680-df33-11eb-950a-48a40b423ad2.png)